BillManagementSystem/accounting/urls.py

```diff
@@ -1,7 +1,10 @@
 from django.urls import path
 
 from . import views
+from strawberry.django.views import GraphQLView
+from .schema.schema import schema
 
 urlpatterns = [
-    path('', views.index, name='index')
+    path('', views.index, name='index'),
+    path('graphql', GraphQLView.as_view(schema=schema)),
```

Comment:
Usually graphql root will be at server root, e.g. `localhost:8000/graphql`. It is an antipattern to nest is like this.